ansible/Ansiblefile

```diff
@@ -2,4 +2,3 @@
 #^syntax detection
  
 site "https://galaxy.ansible.com/api/v1";
- 
```

Comment:
Missing EOF line

Reply:
Not sure why github doesn't show it, but there is an empty line in the file. Try to checkout, you'll see it :)